gsuite/utils.go

```diff
@@ -34,7 +34,7 @@ func retryTime(retryFunc func() error, minutes int) error {
 		if err == nil {
 			return nil
 		}
-		if gerr, ok := err.(*googleapi.Error); ok && (gerr.Errors[0].Reason == "quotaExceeded" || gerr.Code == 404 || gerr.Code == 429 || gerr.Code == 500 || gerr.Code == 502 || gerr.Code == 503) {
```

Comment:
Unfortunately this is not an option- we retry 404's because sometimes it takes a while for Google to create the resource.

Reply:
Perhaps there is a better solution, though I'm not sure what yet.

Reply:
I have added a specific check for the 404 where I was relying on it.